gettsim/benefits/arbeitsl_geld_2/arbeitsl_geld_2.py

```diff
@@ -100,13 +145,25 @@ def regelsatz_m_hh_bis_2010(
     kindersatz_m_hh,
     arbeitsl_geld_2_params,
 ):
+    """
+
+    Parameters
+    ----------
+    anz_erwachsene_hh
+    alleinerziehenden_mehrbedarf_hh
+    kindersatz_m_hh
+    arbeitsl_geld_2_params
+
+    Returns
+    -------
+
+    """
     data = np.where(
         anz_erwachsene_hh == 1,
         arbeitsl_geld_2_params["regelsatz"] * (1 + alleinerziehenden_mehrbedarf_hh),
         arbeitsl_geld_2_params["regelsatz"]
         * (
-            (2 + alleinerziehenden_mehrbedarf_hh)
```

Comment:
I think this is wrong? We do not need to add the mehrbedarf if the household contains both parents? @Eric-Sommer  Our tests don't fail if I delete that! We should have a test case detecting that.

Reply:
Just to confirm, the model can identify the "Mehrbedarf" of single parents only . None of the other additional needs cases could be identified in it (e.g. employable disabled people taking part in labor market reintegration measures)? I understand that all other cases could not be easily identified, just wanted to make sure it is the case. I have only seen it for single parents in the policy functions.